README.md

```diff
@@ -1,11 +1,29 @@
 <a href="https://tiledb.com"><img src="https://github.com/TileDB-Inc/TileDB/raw/dev/doc/source/_static/tiledb-logo_color_no_margin_@4x.png" alt="TileDB logo" width="400"></a>
 
-[![TileDB-SOMA CI](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SOMA/actions/workflows/ci.yml)
+[![TileDB-SingleCell CI](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml/badge.svg)](https://github.com/single-cell-data/TileDB-SingleCell/actions/workflows/ci.yml)
 
-# TileDB-SOMA
+# TileDB-SingleCell
 
 A C++ library with APIs in Python and R for efficient storage and retrieval of single-cell data using [TileDB Embedded][tiledb].
 
+Note this branch, `main`, implements the [newer specfication](https://github.com/single-cell-data/SOMA/blob/main/abstract_specification.md).  Please also see the `main-old` branch which implements the [older specification](https://github.com/single-cell-data/TileDB-SOMA/blob/main-old/spec/specification.md).
+
+# Language bindings and installation instructions
+
+* [Python](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/python)
+* [R](https://github.com/single-cell-data/TileDB-SOMA/tree/main/apis/r)
+* [API documentation](https://tiledb-inc-tiledb-soma.readthedocs-hosted.com/en/latest/)
+  * Note: these document the `main-old` branch; `main` docs are a WIP.
+
+# Issues and contacts
```

Comment:
Recommend an explicit section for reporting security issues, as you usually do not want those filed as an open issue.

Here is what we use for most of our organization's repos:

```
# Reporting Security Issues
Please note: If you believe you have found a security issue, please responsibly disclose it by contacting [security@????.com](mailto:security@????.com).
```

Reply:
thanks @bkmartinjr ! let me find out our internal policy/POC for this

Reply:
updated